mitosheet/mitosheet/tests/public/v3/sheet_functions/date_functions/test_today.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Saga Inc.
+# Distributed under the terms of the GPL License.
+"""
+Contains tests for the TODAY function.
+"""
+
+import pytest
+import pandas as pd
+
+from mitosheet.public.v3.sheet_functions.date_functions import TODAY
+from mitosheet.tests.test_utils import create_mito_wrapper, create_mito_wrapper_with_data
+from datetime import date
+
+def test_today_valid():
+    assert TODAY() == pd.to_datetime(date.today())
+
+def test_today_in_sheet():
```

Comment:
Can we write a test that creates an analysis and replays it that has the TODAY formula in it? Just to make sure it runs -- it should obvi return the same value between runs.